data/networks.ts

```diff
@@ -119,6 +150,74 @@ const publicChains: ZkSyncNetwork[] = [
     l1Network: l1Networks.sepolia,
     hidden: true,
   },
+  {
+    id: 9075,
+    key: "gateway",
+    name: "ZKsync Gateway Mainnet",
+    rpcUrl: "https://rpc.era-gateway-mainnet.zksync.dev",
+    blockExplorerUrl: "https://explorer.era-gateway-mainnet.zksync.dev",
+    blockExplorerApi: "https://block-explorer-api.era-gateway-mainnet.zksync.dev",
+    l1Network: l1Networks.mainnet,
+    displaySettings: {
+      onramp: false,
+      showPartnerLinks: false,
+    },
+    nativeCurrency: {
+      name: "ZKsync",
+      symbol: "ZK",
+      decimals: 18,
+      iconUrl: "img/era.svg",
+      l1Address: "0x66A5cFB2e9c529f14FE6364Ad1075dF3a649C0A5",
+    },
+    nativeBridgingOnly: true,
+    getTokens: () => {
+      return [
+        {
+          address: L2_BASE_TOKEN_ADDRESS,
+          l1Address: "0x66A5cFB2e9c529f14FE6364Ad1075dF3a649C0A5",
+          name: "ZKsync",
+          symbol: "ZK",
+          decimals: 18,
+          iconUrl: "/img/era.svg",
+          isETH: false,
+        },
```

Comment:
No tokens are returned from Explorer API for Gateway so I had to explicitly define the ZK base token. Is there a better way to get this information?